src/sonic-yang-models/yang-models/sonic-interface.yang

```diff
@@ -66,13 +66,19 @@ module sonic-interface {
 					}
                                         default "0";
 				}
+
 				leaf mpls {
 					description "Enable/disable MPLS routing for the interface";
 					type enumeration {
 						enum enable;
 						enum disable;
 					}
 				}
+
+                leaf loopback_action {
+                    description "Packet action when a packet ingress and gets routed on the same IP interface";
+                    type stypes:loopback_action;
```

Comment:
shouldn't we specify the default action?

Reply:
@prsunny In general no. If we specify default action, this will be written to config_db. In case if the action is not implemented by other vendors there would be a SAI failure followed by orchagent crash.
So previously in yang subgroup we decided not to have default action for attributes that would be programmed into ASIC